spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, :type => :controller do
+
```

Comment:
There are some commented-out bits of code in here, not sure if I should delete them or if you're still working on something and need it for reference.